server/tox.ini

```diff
@@ -54,6 +54,14 @@ deps =
 commands =
     mkdocs serve {posargs}
 
+[testenv:buildcli]
+extra =
+    dev
+deps =
+    pyinstaller
+commands =
+    pyinstaller --onefile scripts/cli.py
```

Comment:
This doesn't seem to work correctly since the generated output file doesn't have any options other than `--help` and `--version`. I'm not sure if this still WIP or actually has a bug. I'm suspecting it might be something to do with the `cli.py` file not importing stuff correctly for pyinstaller to use in click.

Also be sure to add `{posargs}` at the end so additional optional flags can be passed.

Reply:
Had to blow away my tox env to reproduce.  `rm -rf .tox && tox -e buildcli && ./dist/cli` .  Something I had before was making this work.